src/passkey_child/passkey_child_common.c

```diff
@@ -121,14 +124,25 @@ parse_arguments(int argc, const char *argv[], struct passkey_data *data)
         case 'a':
             if (data->action != ACTION_NONE
                 && data->action != ACTION_AUTHENTICATE) {
-                fprintf(stderr, "\n--register and --authenticate are mutually exclusive " \
+                fprintf(stderr, "\nActions (i.e. register, authenticate) are mutually exclusive " \
                                 "and should be used only once.\n\n");
                 poptPrintUsage(pc, stderr, 0);
                 ret = EINVAL;
                 goto done;
             }
             data->action = ACTION_AUTHENTICATE;
             break;
+        case 'f':
+            if (data->action != ACTION_NONE
+                && data->action != ACTION_FORMAT_PEM) {
+                fprintf(stderr, "\nActions (i.e. register, authenticate) are mutually exclusive " \
```

Comment:
Maybe it's better to remove `(i.e. register, authenticate)`? It misses `f`.

Reply:
Removed as suggested.